coil-base/src/main/java/coil/collection/SparseIntArraySet.kt

```diff
@@ -16,45 +16,52 @@ import coil.util.growAndInsert
 class SparseIntArraySet @JvmOverloads constructor(initialCapacity: Int = 10) {
 
     private var elements = IntArray(initialCapacity)
-    private var size = 0
+    private var _size = 0
 
-    /** Adds an element to the set. */
+    /** Returns the number of elements that this set currently stores. */
+    val size: Int @JvmName("size") get() = _size
```

Comment:
Does `JvmName` just allow it to be a method for Java callers? Or is it needed otherwise?

Reply:
@Jawnnypoo It sets the callable name in Java so Java callers call `set.size()` instead of `set.getSize()`.